tests/test_launcher.py

```diff
@@ -8,24 +8,30 @@
 from ansys.fluent.core import PyFluentDeprecationWarning  # noqa: F401
 from ansys.fluent.core.exceptions import DisallowedValuesError, InvalidArgument
 from ansys.fluent.core.launcher import launcher_utils
-from ansys.fluent.core.launcher.launcher import create_launcher
-from ansys.fluent.core.launcher.launcher_utils import (
+from ansys.fluent.core.launcher.error_handler import (
     DockerContainerLaunchNotSupported,
-    FluentLinuxGraphicsDriver,
-    FluentMode,
-    FluentUI,
-    FluentWindowsGraphicsDriver,
     GPUSolverSupportError,
     LaunchFluentError,
-    LaunchMode,
     UnexpectedKeywordArgument,
-    _build_fluent_launch_args_string,
-    _build_journal_argument,
-    _is_windows,
     _raise_non_gui_exception_in_windows,
+)
+from ansys.fluent.core.launcher.launcher import create_launcher
+from ansys.fluent.core.launcher.launcher_utils import (
+    _build_journal_argument,
     check_docker_support,
+    is_windows,
+)
+from ansys.fluent.core.launcher.process_launch_string import (
+    _build_fluent_launch_args_string,
     get_fluent_exe_path,
 )
+from ansys.fluent.core.launcher.pyfluent_enums import (
+    FluentLinuxGraphicsDriver,
+    FluentMode,
+    FluentUI,
```

Comment:
**Aside** Just looking at this now, which we settled on by committee. It's non-obvious IMHO, and as type name, you'd think it actually represents a UI itself. Compare with FluentMode and LaunchMode which are explicitly modes and therefore much clearer. @mkundu1 @raph-luc

Reply:
Sounds right @seanpearsonuk. Thank you.